src/utilities/dump_flags/DumpFlags_processor.h

```diff
@@ -54,7 +54,7 @@ class DumpFlags_processor: public JEventProcessor, public eicrecon::SpdlogMixin<
     std::string m_json_file_name = "";
 
     /// Print parameter summary to screen at end of job
-    bool m_print_to_screen = false;
+    bool m_print_to_screen = true;
```

Comment:
Ughh. Why turn this back on? I found it highly annoying to have to scroll past pages of parameters when trying to see useful messages for debugging. This is why I set the default to off. Maybe we could dump this to a config file by default instead? If you find it that useful to have it print to the screen we can leave it on. I was just loving it being disabled by default though. Just my 2 cents.